storage-serviceaccount-cmdline-sample/README.md

```diff
@@ -26,28 +26,28 @@ You may need to set your `JAVA_HOME`.
         git clone https://github.com/GoogleCloudPlatform/cloud-storage-docs-xml-api-examples.git
         cd cloud-storage-docs-xml-api-examples/storage-serviceaccount-cmdline-sample
 
-2. Update key.p12 file.
-3. Edit `StorageServiceAccountSample.java` to add your bucket and service account email.
+2. Update key.json file.
+3. Edit `StorageServiceAccountSample.java` to add your bucket.
```

Comment:
How about to make the bucket name either of environment variable, or command line argument so that you don't have to edit the java file?